hooks/dashboard.ts

```diff
@@ -28,3 +37,29 @@ export const useGetTempSavedList = (blogUrl: string) => {
   const { data } = useQuery([QUERY_KEY_DASHBOARD.getTempSavedList], () => getTempSavedList(blogUrl));
   return data;
 };
+
+export const usePostCategory = (blogUrl: string, name: string, description: string) => {
+  const queryClient = useQueryClient();
+
+  const mutation = useMutation([QUERY_KEY_DASHBOARD.postCategory], () => postCategory(blogUrl, name, description), {
+    onSuccess: () => {
+      queryClient.invalidateQueries([QUERY_KEY_DASHBOARD.getCategoryList]);
+    },
+  });
+  return mutation;
+};
+
```

Comment:
```suggestion
export const usePostCategory = (blogUrl: string, name: string, description: string) => {
  const queryClient = useQueryClient();

  return useMutation([QUERY_KEY_DASHBOARD.postCategory], () => postCategory(blogUrl, name, description), {
    onSuccess: () => {
      queryClient.invalidateQueries([QUERY_KEY_DASHBOARD.getCategoryList]);
    },
  });
};

```
요롷게 어때요?!

Reply:
```suggestion export const usePostCategory = (blogUrl: string, name: string, description: string) => {
  const queryClient = useQueryClient();
  return useMutation(() => postCategory(blogUrl, name, description), {
    onSuccess: () => {
      queryClient.invalidateQueries([QUERY_KEY_DASHBOARD.getCategoryList]);
    },
  });
};

```
useMutation key 없앨거면?!